extractor/filesystem/list/list.go

```diff
@@ -134,6 +135,7 @@ var (
 	// Rust extractors.
 	Rust []filesystem.Extractor = []filesystem.Extractor{
```

Comment:
We made some refactors to how list.go is structured in https://github.com/google/osv-scalibr/commit/99ab001bd6ab7f3fd986d9a33f5aa1da30e18f92 so this file will need to be synced